llvm/unittests/Support/ErrorTest.cpp

```diff
@@ -1132,4 +1132,26 @@ TEST(Error, moveInto) {
   }
 }
 
+TEST(Error, FatalBadAllocErrorHandlersInteraction) {
+  auto ErrorHandler = [](void *Data, const char *, bool) {};
+  install_fatal_error_handler(ErrorHandler, nullptr);
+  // The following call should not crash
```

Comment:
I think you should expand just a little more your comment, just to explain we're doing this sequence of calls because of a bug that happened previously inside `install_bad_alloc_error_handler`.